I remember running into

```
  CXX      libbitcoin_node_a-blockencodings.o
In file included from blockencodings.cpp:15:
In file included from ./txmempool.h:9:
In file included from ./coins.h:10:
In file included from ./core_memusage.h:10:
In file included from ./memusage.h:10:
./support/allocators/pool.h:161:27: error: aligned allocation function of type 'void *(std::size_t, std::align_val_t)' is only available on macOS 10.13 or newer
        void* storage = ::operator new (m_chunk_size_bytes, std::align_val_t{ELEM_ALIGN_BYTES});
                          ^
./support/allocators/pool.h:161:27: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
./support/allocators/pool.h:204:15: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.13 or newer
            ::operator delete ((void*)chunk, std::align_val_t{ELEM_ALIGN_BYTES});
              ^
./support/allocators/pool.h:204:15: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
./support/allocators/pool.h:235:18: error: aligned allocation function of type 'void *(std::size_t, std::align_val_t)' is only available on macOS 10.13 or newer
        return ::operator new (bytes, std::align_val_t{alignment});
                 ^
./support/allocators/pool.h:235:18: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
./support/allocators/pool.h:250:15: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.13 or newer
            ::operator delete (p, std::align_val_t{alignment});
              ^
./support/allocators/pool.h:250:15: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
4 errors generated.
```

on a local x86_64-darwin machine the other day, too. I think it's an issue with NixOS clang - though I haven't looked into it any deeper.

--- a/pkgs/applications/blockchains/bitcoin/default.nix
+++ b/pkgs/applications/blockchains/bitcoin/default.nix
@@ -33,14 +33,14 @@ let
 in
 stdenv.mkDerivation rec {
   pname = if withGui then "bitcoin" else "bitcoind";
-  version = "25.1";
+  version = "26.0";
 
   src = fetchurl {
     urls = [
       "https://bitcoincore.org/bin/bitcoin-core-${version}/bitcoin-${version}.tar.gz"
     ];
     # hash retrieved from signed SHA256SUMS
-    sha256 = "bec2a598d8dfa8c2365b77f13012a733ec84b8c30386343b7ac1996e901198c9";
+    sha256 = "ab1d99276e28db62d1d9f3901e85ac358d7f1ebcb942d348a9c4e46f0fcdc0a1";
   };
 
   nativeBuildInputs =
@@ -55,9 +55,9 @@ stdenv.mkDerivation rec {
     ++ lib.optionals withGui [ qrencode qtbase qttools ];
 
   postInstall = ''
-    installShellCompletion --cmd bitcoin-cli --bash contrib/completions/bash/bitcoin-cli.bash-completion
-    installShellCompletion --cmd bitcoind --bash contrib/completions/bash/bitcoind.bash-completion
-    installShellCompletion --cmd bitcoin-tx --bash contrib/completions/bash/bitcoin-tx.bash-completion
+    installShellCompletion --bash contrib/completions/bash/bitcoin-cli.bash
+    installShellCompletion --bash contrib/completions/bash/bitcoind.bash
+    installShellCompletion --bash contrib/completions/bash/bitcoin-tx.bash
 
     installShellCompletion --fish contrib/completions/fish/bitcoin-cli.fish
     installShellCompletion --fish contrib/completions/fish/bitcoind.fish

